torch/ao/quantization/backend_config/backend_config.py

```diff
@@ -85,11 +85,11 @@ class DTypeConfig:
     Config for the set of supported input/output activation, weight, and bias data types for the
     patterns defined in :class:`~torch.ao.quantization.backend_config.BackendConfig`.
     """
-    input_dtype: Optional[DTypeWithConstraints] = None
-    output_dtype: Optional[DTypeWithConstraints] = None
-    weight_dtype: Optional[DTypeWithConstraints] = None
-    bias_dtype: Optional[torch.dtype] = None
-    is_dynamic: Optional[bool] = None
+    input_dtype: DTypeWithConstraints
+    output_dtype: DTypeWithConstraints
+    weight_dtype: DTypeWithConstraints
+    bias_dtype: Optional[torch.dtype]
+    is_dynamic: Optional[bool]
```

Comment:
btw, Optional[bool] also doesn't make sense, False in bool can represent the non-valid case I think, but it can be cleaned up in a separate PR

Reply:
Ok, this was already there before this PR so let's clean up separately